xcodeproject/xcodeproj/xcodeproj_test.go

```diff
@@ -389,15 +389,10 @@ func TestTargets(t *testing.T) {
 		target, ok := project.Proj.Target("7D0342D720F4B5AD0050B6A6")
 		require.True(t, ok)
 
-		dependentTargets := target.DependentTargets()
+		dependentTargets := project.DependentTargetsOfTarget(target)
 		require.Equal(t, 2, len(dependentTargets))
 		require.Equal(t, "WatchKitApp", dependentTargets[0].Name)
 		require.Equal(t, "WatchKitApp Extension", dependentTargets[1].Name)
-
```

Comment:
Deleted this asserts because `DependentExecutableProductTargets()` is removed. Also, these asserts were testing the same as the section above.